saleor/graphql/page/types.py

```diff
@@ -218,25 +246,57 @@ def resolve_content_json(root: ChannelContext[models.Page], _info: ResolveInfo):
         content = root.node.content
         return content if content is not None else {}
 
-    @staticmethod
-    def resolve_attributes(root: ChannelContext[models.Page], info: ResolveInfo):
+    @classmethod
+    def resolve_assigned_attributes(
+        cls,
+        root: ChannelContext[models.Page],
+        info: ResolveInfo,
+        limit: int = DEFAULT_NESTED_LIST_LIMIT,
+    ):
+        return cls._resolve_assigned_attributes(root, info, limit)
+
+    @classmethod
+    def resolve_assigned_attribute(
+        cls, root: ChannelContext[models.Page], info: ResolveInfo, slug: str
+    ):
+        return cls._resolve_assigned_attribute(root, info, slug)
+
+    @classmethod
+    def resolve_attributes(cls, root: ChannelContext[models.Page], info: ResolveInfo):
+        return cls._resolve_assigned_attributes(root, info)
```

Comment:
How about moving `cls._resolve_assigned_attributes` to `resolvers.py` like in other cases?